platforms/android/app/src/main/java/com/maq/xprize/cci/hindi/SplashScreenActivity.java

```diff
@@ -65,21 +70,35 @@ public void onRequestPermissionsResult(int requestCode,
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
-        // call main activity if extraction done
-        String flagFilePath = "/storage/emulated/0/Android/data/com.maq.xprize.CCI.hindi/files/.success.txt";
+        sharedPref = getSharedPreferences("ExpansionFile", MODE_PRIVATE);
+        String flagFilePath = "/storage/emulated/0/Android/data/" + getPackageName() + "/files/.success.txt";

```

Comment:
We can remove the flag file dependency since we are using shared preferences.

Reply:
Yes. Added in the backlog